src/routes/docs/configuring/keyboard-shortcuts/+page.md

```diff
@@ -11,6 +11,7 @@ Files comes with built in keyboard shortcuts to make navigating the app easier.
 | <kbd>Ctrl</kbd> + <kbd>Z</kbd>                      | Undo operation |
 | <kbd>Ctrl</kbd> + <kbd>Y</kbd>                      | Redo operation |
 | <kbd>Ctrl</kbd> + <kbd>T</kbd>                      | Create new tab |
+| <kbd>Ctrl</kbd> + <kbd>Enter</kbd>                  | Open selected folders in new tab |
```

Comment:
@yaichenbaum Is this shortcut accurate and has it shipped to the store yet?

Reply:
In the Preview 2.3.52 for now.

Reply:
Thanks, LGTM